include/deal.II/base/mpi.h

```diff
@@ -415,6 +467,85 @@ namespace Utilities
         return MPI_LONG_DOUBLE;
       }
 #endif
+
+      template <typename T>
+      inline
+      T op (const MPI_Op &mpi_op,
+            const T &t,
+            const MPI_Comm &mpi_communicator)
```

Comment:
since the function does a reduction, maybe calling it something different than `op` might be more descriptive :-)